locallib.php

```diff
@@ -0,0 +1,51 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Tool_pluginskel internal functions.
+ *
+ * @package     tool_pluginskel
+ * @copyright   2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace tool_pluginskel\local\util;
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * Expand the path for $path.
+ *
+ * @param string $path The path to be expanded.
+ * @return string | false The expanded path, false on failure.
+ */
+function expand_path($path) {
```

Comment:
I assumed that since the function is in the plugin namespace it doesn't need to have the full frankenstyle prefix, just like classes.

But sure, I'll modify the function name.